source/partials/_nav.slim

```diff
@@ -5,20 +5,21 @@
   = nav_overlay_item('Apprenticeship', '/apprenticeship')
   = nav_overlay_item('Blog', '/blog')
 
-nav#main-nav.Nav.Nav--topPositioned class="#{current_page.data.nav_class}" aria-label=('Main navigation')
-  = link_to '/', class: 'Nav-logo'
-    .NavLogo class=nav_logo_class
+header
+  #main-nav.Nav.Nav--topPositioned class="#{current_page.data.nav_class}"
+    = link_to '/', "aria-label" => "Homepage", class: 'Nav-logo'
+      .NavLogo class=nav_logo_class
 
-  .Nav-menu
-    .Burger class=nav_burger_class
-      span.Burger-line
-      span.Burger-line
-      span.Burger-line
+    .Nav-menu
+      .Burger class=nav_burger_class
+        span.Burger-line
+        span.Burger-line
+        span.Burger-line
 
-  ul.Nav-items
-    = nav_item('Work', '/work')
-    = nav_item('Company', '/company')
-    = nav_item('Community', '/community')
-    = nav_item('Apprenticeship', '/apprenticeship')
-    = nav_item('Blog', '/blog')
-.u-navPlaceholder
+    nav aria-label='Main navigation' role='navigation'
```

Comment:
It's just that you'll need a 3 level deep structure:
`nav > ul > li`.
Many times had I the need to do some CSS extra work just to make the whole thing work due to the extra level.
Not sure the `list` brings anything semantically speaking since the nav is already self explanatory. That's my opinion, not a `de facto` answer.